kerl

```diff
@@ -411,8 +411,10 @@ assert_perl()
 
 get_javac_version()
 {
-    java_loc=$(which javac)
-    if [ -z "$java_loc" ]; then
+    # http://apple.stackexchange.com/a/135079
+    /usr/libexec/java_home >/dev/null 2>&1
```

Comment:
Does this hold up on other systems? It doesn't look like it works on my Ubuntu 14.04 server, at least - or is this function only called on MacOS?